src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
@trevnorris how would you name it?

Just clarifying why this is - It needs to run after all microtasks have finished running. This is so arbitrarily nested native promises have gotten a change to schedule and run.